webpack/InsightsHostDetailsTab/InsightsTabActions.js

```diff
@@ -0,0 +1,30 @@
+import API from 'foremanReact/API';
+import { insightsCloudUrl } from '../InsightsCloudSync/InsightsCloudSyncHelpers';
+import {
+  INSIGHTS_HITS_REQUEST,
+  INSIGHTS_HITS_SUCCESS,
+  INSIGHTS_HITS_FAILURE,
+} from './InsightsTabConstants';
+
+export const fetchHits = hostname => async dispatch => {
+  try {
+    dispatch({
+      type: INSIGHTS_HITS_REQUEST,
+      payload: {},
```

Comment:
I am not sure why do we need to emit the request, but if we already do it, I suppose it should have `hostname` (or `hostId`) parameter as the payload.

Reply:
in a follow up PR it will be used to handle better pending state etc..
for now I just created its structure